.travis.yml

```diff
@@ -111,6 +111,28 @@ install:
 script:
     - export BASE_PATH=$PATH
     ###
+    ### Build without transport
+    ###
+    - cd $SOS_SRC
+    - mkdir no-transport-build
+    - cd no-transport-build
+    - export PATH=$TRAVIS_INSTALL/hydra/bin:$BASE_PATH
+    - ../configure --prefix=$TRAVIS_INSTALL/sandia-shmem-none $SOS_GLOBAL_BUILD_OPTS $SOS_BUILD_OPTS
+    - make $TRAVIS_PAR_MAKE
+    - make $TRAVIS_PAR_MAKE check TESTS=
+    - make install
+    ###
+    ### Build with only CMA
+    ###
+    - cd $SOS_SRC
+    - mkdir cma-build
+    - cd cma-build
+    - export PATH=$TRAVIS_INSTALL/hydra/bin:$BASE_PATH
+    - ../configure --prefix=$TRAVIS_INSTALL/sandia-shmem-cma $SOS_GLOBAL_BUILD_OPTS $SOS_BUILD_OPTS
```

Comment:
Good catch!  XPMEM requires a kernel patch, so it would be hard to test in Travis.  You could potentially try to fake the user space part of XPMEM without the kernel patch to exercise the SOS code, but it would be a hack.  The most practical option is probably to test the XPMEM code on a local cluster that is patched or by hand on an XC.  The better option would be to have this run nightly somewhere..